programs/stopwatch_example/Makefile

```diff
@@ -0,0 +1,10 @@
+CFLAGS=-Wall -Werror -pedantic -std=gnu99
```

Comment:
This isn't 1989 or CSC 357 :joy:, so we can drop `-pedantic` and use newer C features!